datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -435,7 +423,7 @@ impl FileOpener for ParquetOpener {
             // Row group pruning: attempt to skip entire row_groups
             // using metadata on the row groups
             let file_metadata = builder.metadata();
-            let row_groups = prune_row_groups(
+            let row_groups = row_groups::prune_row_groups(
```

Comment:
The whole point of the PR is to move this function (and its tests) into its own module